app-store-connect/src/profile_api.rs

```diff
@@ -191,7 +213,7 @@ pub struct ProfileAttributes {
     pub platform: String,
     pub profile_content: String,
     pub uuid: String,
-    pub created_date: String,
+    pub created_date: Option<String>,
```

Comment:
I guess you encountered a missing / null value in the wild and that's why this change was made?

Reply:
that's correct